pom.xml

```diff
@@ -1256,6 +1248,7 @@
 		<hibernateVersion>5.6.15.Final</hibernateVersion>
 		<hibernateSearchVersion>6.2.4.Final</hibernateSearchVersion>
 		<luceneVersion>8.11.2</luceneVersion>
+		<infinispanVersion>13.0.22.Final</infinispanVersion>
```

Comment:
Did we use this version instead of `15.2.1.Final` because of hibernate/spring version compatibility problems?